test/integration/users_signup_test_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class UsersSignupTest < ActionDispatch::IntegrationTest
+
+  test "invalid signup information" do
+    get signup_path
+    assert_no_difference 'User.count' do
+      post users_path, user: {
+                                name: "",
+                                email: "user@invalid",
+                                password: "foo",
+                                password_confirmation: "bar"
+                              }
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Indent the right brace the same as the start of the line where the left brace is.](https://www.codacy.com/app/conta-srdr/forsythia/file/2129755518/issues/source?bid=3139784&fileBranchId=3147671#l13)